ert_gui/simulation/run_dialog.py

```diff
@@ -191,6 +194,36 @@ def on_new_iteration(self, parent: QModelIndex, start: int, end: int) -> None:
 
             self._tab_widget.addTab(widget, f"Iteration {iter}")
 
+    @Slot(QModelIndex)
+    def _job_clicked(self, index):
+        node = index.internalPointer()
+        if node is None or node.type != NodeType.JOB:
+            return
+
+        parent = node.parent
+        while parent.type != NodeType.ROOT:
+            if parent.type == NodeType.REAL:
+                real = parent.id
+            elif parent.type == NodeType.ITER:
+                iter = parent.id
+            parent = parent.parent
+
+        if index.column() == 4:
+            selected_file = node.data.get("stdout")
```

Comment:
if the index is at that column, `node.data.get("stdout")` should equate to `index.data()` (80% sure)

Reply:
index.data() gives the displayed data apparently, i.e. "OPEN"

Reply:
Okay, then you could add a role and do `index.data(FileRole)` or somesuch.

Reply:
The idea is that the model is a proxy on top of the underlying data structure